js/plugins/verticalLine.js

```diff
@@ -1,66 +1,105 @@
-/*
- * Handles the drawing of a vertical line
- * on hover over the timeline in the simple view
+/**
+ * Plugin to handle the drawing of a vertical line 
+ * on hovering over the timeline
+ * @type {clinicalTimelinePlugin}
  */
-var clinicalTimelineVerticalLine = function(enableVerticalLine, beginning, ending, advancedView, tooltipOnVerticalLine) {
-  if (enableVerticalLine) {
-    var hoverLineGroup = d3.select(".timeline").append("g")
-      .attr("class", "hover-line");
-  
-    var hoverLine = hoverLineGroup
-      .append("line")
-      .attr("x1", 0)
-      .attr("x2", 0) 
-      .attr("y1", 20)
-      .attr("y2", 268)
-      .style("stroke", "#ccc");
+var clinicalTimelineVerticalLine = new clinicalTimelinePlugin("verticalLine", "Vertical Timeline", {tooltipControllerId: "#tooltip-controller", hoverBegin: 200, hoverEnd: 770});
```

Comment:
The `#tooltip-controller` is now specified in the library's JS. How would one instantiate the clinicalTimelineVerticalLine plugin with a different `tooltipControllerId` from `index.html`?